core/iwasm/common/gc/gc_object.c

```diff
@@ -189,6 +189,20 @@ wasm_struct_obj_get_field(const WASMStructObjectRef struct_obj,
     }
 }
 
+uint32
+wasm_struct_obj_get_field_count(const WASMStructObjectRef struct_obj)
+{
+    uint32 field_count = 0;
+
+    WASMRttTypeRef rtt_type =
+        (WASMRttTypeRef)wasm_object_header((WASMObjectRef)struct_obj);
+    WASMStructType *struct_type = (WASMStructType *)rtt_type->defined_type;
+    if (struct_type->field_count > 0)
+        field_count = struct_type->field_count;
+
+    return field_count;
```

Comment:
How about `return struct_type->field_count` direclty and removing `uint32 field_count = 0`?

Reply:
OK done